contracts/contracts/GrantRoundManager.sol

```diff
@@ -18,22 +18,16 @@ contract GrantRoundManager {
   /// @notice Address of the GrantRegistry
   GrantRegistry public immutable registry;
 
-  /// @notice Address of the Uniswap V3 Router used for token swaps
-  ISwapRouter public immutable router;
-
   /// @notice Address of the ERC20 token in which donations are made
   IERC20 public immutable donationToken;
 
-  /// @notice Used during donations to temporarily store swap output amounts
   mapping(IERC20 => uint256) internal swapOutputs;
 
-  /// @notice Used during donations to temporarily store donation ratio sums to ensure totals are always 100%
   mapping(IERC20 => uint256) internal donationRatios;
 
   /// @notice WETH address
-  IERC20 public constant WETH = IERC20(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);
+  IERC20 public WETH;
 
-  /// @notice Scale factor
```

Comment:
would we wanna still keep this / document it better @mds1 ?

Reply:
Oh yea, we should keep this comment. Maybe we phrase it as

```
/// @notice Scale factor on percentages when constructing `Donation` objects. One WAD represents 100%
```

Reply:
was getting some errors with the notice tags after changing compiler versions: ```contracts/GrantRoundManager.sol:28:3: DocstringParsingError: Documentation tag @notice not valid for non-public state variables.```

Reply:
Ah sorry, yes that makes sense, sorry should be `@dev` then.

`@notice` is used to document public-facing methods/variables, and `@dev` for internal methods/variables (or for additional additional developer-related info somewhere)